coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ServerResource.java

```diff
@@ -171,10 +176,17 @@ public Response<Map<String, Integer>> getNodeStatusTotal() {
     return execute(
         () -> {
           ClusterManager clusterManager = getClusterManager();
+          List<ServerNode> excludeNodes =
+              clusterManager.getExcludeNodes().stream()
+                  .map(exclue -> new ServerNode(exclue))
+                  .collect(Collectors.toList());
           Map<String, Integer> stringIntegerHash =
-              Stream.concat(
-                      clusterManager.getServerList(Collections.emptySet()).stream(),
-                      clusterManager.getLostServerList().stream())
+              Stream.of(
+                      clusterManager.list(),
+                      clusterManager.getLostServerList(),
+                      excludeNodes,
+                      clusterManager.getUnhealthyServerList())
```

Comment:
`clusterManager.list()` contains all of unhealthy nodes and part of the excluded nodes, so, we may need use `distinct()`

Reply:
> `clusterManager.list()` contains all of unhealthy nodes and part of the excluded nodes, so, we may need use `distinct()`

Have been modified.